axis.go

```diff
@@ -280,12 +280,10 @@ func (a *verticalAxis) size() (w vg.Length) {
 // draw draws the axis along the left side of a draw.Canvas.
 func (a *verticalAxis) draw(c draw.Canvas) {
 	x := c.Min.X
+	a.Label.TextStyle.Rotation += math.Pi / 2.
```

Comment:
Agreed. We can change line 286 to:
```
sty := a.Label.TextStyle
sty.Rotation += math.Pi/2
c.FillText(sty, x, c.Center().Y, -0.5, 0, a.Label.Text)
```
Otherwise, everyone would need to manually set the rotation of the label text each time, which would be annoying.